docs/intro.md

```diff
@@ -35,9 +35,12 @@ Afterwards, you will only need to switch your clients to connect to Percona Serv
 
 ## Filtered replication
 
-You can replicate the whole dataset or only specific subset of data. This is called a filtered replication. You can use it for various use cases. For example, to spin up a new development environment with a specific subset of data instead of the whole dataset. Or to optimize cloud storage costs for hybrid environments where your target MongoDB deployment runs in the cloud.
+You can replicate the whole dataset or only specific subset of data. Such replication is called a filtered replication. You can use it for various use cases, such as:
```

Comment:
You can replicate either an entire dataset or only a subset of the data, which is a filtered replication. You can use a filter replication in the following cases: